src/components/questionStructure/Task/variants/handwritingTasks/components/ViewOnlyCanvas.tsx

```diff
@@ -0,0 +1,44 @@
+import { Excalidraw } from '@excalidraw/excalidraw'
+import { ExcalidrawElement } from '@excalidraw/excalidraw/types/element/types'
+import { ExcalidrawImperativeAPI } from '@excalidraw/excalidraw/types/types'
+import { Box, Card } from '@radix-ui/themes'
+import React, { useEffect, useState } from 'react'
+
+import './viewOnlyCanvas.scss'
+
+interface ViewOnlyCanvasProps {
+  initialData: readonly ExcalidrawElement[]
+  minHeight?: string
+}
+
+export const ViewOnlyCanvas: React.FC<ViewOnlyCanvasProps> = ({
+  initialData,
+  minHeight = '200px',
```

Comment:
It should have been used in the raw handwriting editor, since its preview box needs to be taller as it could contain lots of writing. This has been done in d7ad68cca69fe0cb074d31d1ed66bc554aa1ddac